HWIMO-BUILD

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# HWIMO-BUILD
+
+#
+# Build debian package for all images created by on-imagebuilder
+#
+set -e
+output_path=/tmp/on-imagebuilder
+
+#build static files
+sudo ./build_all.sh
+
+#prepare workplace for deb build
+rm -rf packagebuild
+git clone . packagebuild
+cd packagebuild
+
+mkdir common
+mkdir pxe
+
+#copy static files
+sudo cp $output_path/builds/* common/
+sudo cp $output_path/syslinux/* pxe/
+sudo cp $output_path/ipxe/* pxe/
+
+#generate package version
+GITCOMMITDATE=$(git show -s --pretty="format:%ci")
+DATESTRING=$(date -d "$GITCOMMITDATE" -u +"%Y-%m-%d-%H%M%SZ")
+
+PKG_VERSION="$DATESTRING"
+if [ -n "$BUILD_NUMBER" ]
+then
+  PKG_VERSION="${PKG_VERSION}-${BUILD_NUMBER}"
+fi
+
+export DEBEMAIL="hwimo robots <hwimo@hwimo.lab.emc.com>"
+export DEBFULLNAME="The HWIMO Robots"
+
+# build files in packagebuild into a debian package 
+dch -v ${PKG_VERSION} autobuild
```

Comment:
Yes, format and function of this ```HWIMO-BUILD``` here is totally the same with that in other repos. IMO HWIMO-BUILD is just to build a deb ignore version information. 
Should anyone run HWIMO-BUILD and build a deb with correct version?
If so I'll add this change to HWIMO-BUILD in all repos.